optimum/amd/ryzenai/utils.py

```diff
@@ -25,3 +30,56 @@ def validate_provider_availability(provider: str):
         raise ValueError(
             f"Asked to use {provider} as an ONNX Runtime execution provider, but the available execution providers are {available_providers}."
         )
+
+
+def plot_bbox(image, detections, output_path="plot_bbox_output.png"):
```

Comment:
I kept re-implementing it, so having a simple import saves time. While it could be in an example script, having it as a utility is more convenient for frequent use, especially since Ryzen has CV models that need visualization.